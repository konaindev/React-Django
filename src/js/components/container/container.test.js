import Container from "./index";
import renderer from "react-test-renderer";

describe("Container", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Container>I'm inside {`<Container />`}.</Container>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
