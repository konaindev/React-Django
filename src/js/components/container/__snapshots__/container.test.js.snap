// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
<div
  className="container"
>
  I'm inside 
  &lt;Container /&gt;
  .
</div>
`;
